drivers/include/ina3221.h

```diff
@@ -0,0 +1,823 @@
+/**
+ * @brief Disable alert callback and argument for alert @p alert
+ *
+ * @param[in,out] dev       Device handle
+ * @param[in] alert         Alert index
+ *
+ * @return INA3221_OK, on success
+ * @return -ERANGE, if @p alert was out of bounds
+ * @return -ENUTSUP, if alert pin was initialized with GPIO_UNDEF
+ */
+int ina3221_disable_alert(ina3221_t *dev, ina3221_alert_t alert);
+#endif
+
+/**
+ * @brief Write configuration register value to configuration register
+ *
+ * @param[in,out]   dev Device handle
+ * @param[in]       cfg Configuration register value in host byte order
+ *
+ * @return      INA3221_OK, on success
+ * @return      -INA3221_I2C_ERROR, if I2C bus acquirement failed
+ * @return      @see i2c_write_regs
+ */
+int ina3221_set_config(ina3221_t* dev, uint16_t cfg);
+
+/**
+ * @brief Read currently saved configuration register value
+ * 
+ * @param[in]       dev Device handle
+ * @param[out]      cfg Pointer to configuration register output value in host byte order
+ * 
+ * @return      INA3221_OK
+ */
+int ina3221_get_config(const ina3221_t* dev, uint16_t* cfg);
+
+/**
+ * @brief Enable channels
+ * 
+ * @param[in,out]   dev Device handle
+ * @param[in]       ech Channel enable flags
+ * 
+ * @return      INA3221_OK, on success
+ * @return      -INA3221_I2C_ERROR, if I2C bus acquirement failed
+ * @return      @see i2c_read_regs
+ * @return      @see i2c_write_regs
+ */
+int ina3221_set_enable_channel(ina3221_t *dev, ina3221_enable_ch_t ech);
+
+/**
+ * @brief Read which channels are currently enabled
+ * 
+ * @param[in]       dev Device handle
+ * @param[out]      ech Pointer to enabled channels output variable
+ * 
+ * @return      Number of enabled channels
+ */
+int ina3221_get_enable_channel(const ina3221_t *dev, ina3221_enable_ch_t* ech);
+
+/**
+ * @brief Update number of samples and write to configuration register
+ * 
+ * @param[in, out]  dev Device handle
+ * @param[in]       ns Number of samples
+ * 
+ * @return      INA3221_OK, on success
+ * @return      -INA3221_I2C_ERROR, if I2C bus acquirement failed
+ * @return      @see i2c_read_regs
+ * @return      @see i2c_write_regs
+ */
+int ina3221_set_num_samples(ina3221_t *dev, ina3221_num_samples_t ns);
+
+/**
+ * @brief Read currently applied number of samples
+ * 
+ * @param[in]       dev Device handle
+ * @param[out]      ns Pointer to number of samples output variable
+ * 
+ * @return  INA3221_OK
+ */
+int ina3221_get_num_samples(const ina3221_t *dev, ina3221_num_samples_t* ns);
+
+/**
+ * @brief Update conversion time of bus voltage ADC and write to configuration register
+ * 
+ * @param[in,out]   dev Device handle
+ * @param[in]       ctb Bus voltage conversion time
+ * 
+ * @return      INA3221_OK, on success
+ * @return      -INA3221_I2C_ERROR, if I2C bus acquirement failed
+ * @return      @see i2c_read_regs
+ * @return      @see i2c_write_regs
+ */
+int ina3221_set_conv_time_badc(ina3221_t* dev, ina3221_conv_time_badc_t ctb);
```

Comment:
How about `bus_adc` instead of `badc`, as ADC is a pretty common abbreviation known by most readers, but BADC is not.